index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" type="text/css" href="css/element_style.css">
+    <link href="https://fonts.googleapis.com/css?family=Quicksand:400,500,700" rel="stylesheet">
+    <link rel="stylesheet" href="lib/owl-carousel/owl.carousel.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <title>Duck On The Run: Home</title>
+</head>
+
+<body>
+    <main id="main_block">
+        <header class="global_header">
+            <div class="header_navigation">
+                <nav>
+                    <ul>
+                        <li><img src="images/more-menu.png" alt="more-button" class="more_menu_button menu" /></li>
+                        <li><a class="menu" href="index.html"><b>How to  Play</b></a></li>
+                        <li><b class="menu" id="text-seperator">|</b></li>
+                        <li><a class="menu" href="#"> <b>Contact Us</b></a></li>
+                        <li><a href="#"></a><img src="images/logo.png" alt="logo-image" class="logo_image"></a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <section>
+            <div class="home_banner">
+                <a href="#"><img src="images/help-btn.png" class="help_button"/></a>
+                <img src="images/dotr-title.png" class="dotr_title" />
+                <article class="description">Connecting numerical learning and research through gameplay</article>
+                <a href="#"><img src="images/googleplay-btn.png" class="app_download_playstore"></a>
+                <a href="#"><img src="images/appstore-btn.png" class="app_download_appstore"></a>
+                <img src="images/home-bannerBg.png" class="home_banner" />
```

Comment:
@appucyriac 
Dont use same class to the parent div and image. Make the image as background to set as background for the banner section.